src/20_Components/amp-selector.html

```diff
@@ -14,138 +14,181 @@
   <script async src="https://cdn.ampproject.org/v0.js"></script>
   <!-- ## Setup -->
   <!--
-    Import the amp-selector component.
+    Import the `amp-selector` component.
   -->
   <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
   <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
```

Comment:
should we add why we are adding those additional libraries?

Reply:
done